templates/pages/about.html.twig

```diff
@@ -26,6 +26,9 @@
         <li>Symfony version: <code>{{ symfony }}</code></li>
         <li>Operating System: <code>{{ os_name }} - <small>{{ os_version }}</small></code></li>
         <li>Assets version: <code id="assetsVersion">-</code></li>
+        <li>Upload filesize limit: <code>{{ config.maxUpload|format_bytes }}</code>
```

Comment:
Can we word this `File size upload limit`?

Reply:
```suggestion
        <li>File size upload limit: <code>{{ config.maxUpload|format_bytes }}</code>
```

Reply:
Done! 👍